app-backend/database/src/main/scala/com/azavea/rf/database/tables/Scenes.scala

```diff
@@ -324,6 +324,17 @@ object Scenes extends TableQuery(tag => new Scenes(tag)) with LazyLogging {
     }
   }
 
+  def sceneNameExistsForUser(name: String, user: User)(implicit database: DB) = {
```

Comment:
Hm, it's an exist function, it makes sense to expect here `Bool` as a return type, what do you think? 

```scala
database.db.run {
  (for {
    scenes <- Scenes
    if scenes.owner === user.id
    if scenes.name === name
  } yield scenes).nonEmpty
}
```